src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -525,11 +527,25 @@ private double calculateOverageWeight(double firstValue, double secondValue, dou
      * time we enter this edge, whereas in a reverse search we get the speed based on the time we exit
      * the edge.
      */
+    public double getVooomSpeed(long timeMillis) {
+
+        if (this.getTimes()!=null) {
+            int i = Collections.binarySearch(this.getTimes(), QueryData.QueryNaw());
```

Comment:
Tutaj trzeba użyć `timeMillis`, a nie `now`

Reply:
@fenix7 mógłbyś nie ustawiać na `resolved` komentarzy i uwag, których nie naniosłeś? Tutaj nadal jest użyte `QueryNaw` zamiast `timeMillis`. To jest kluczowe, bo co nam po lokalnym czasie komputera na którym jest robione zapytanie, skoro nas interesuje czas, w którym będziemy w danym miejscu jechać autem.

Reply:
sury bylem pewny ze to  także odpowiednie wywołanie   teraz  już wywołane

Reply:
teraz juz  resolved